lookbook/docs/patterns/15-colors.md.erb

```diff
@@ -68,6 +68,151 @@ The following variables of Primer are overwritten with the `--accent-color`:
 
 When implementing colors in the application, we should ensure compliance with Primer's [accessibility standards](https://primer.style/foundations/color/accessibility).
 
+## Attributes colors
+
+We are following the same approach GuitHub is doing for their labels. For all configured colors, based on their hex codes and the selected mode, new values will be generated. The highlighting color values should be always visible and readable for:
+- Font color
+- Complete row highlighting
+- Card highlighting
+- Status selector
+- Highlighting via bubble
+
+There is a method that decompose the color hex code into the rgb and hsl values. While highlighting an attribute, this method should be called first:
```

Comment:
That sounds like the developer explicitly has to call these methods to get it work, but that is not true.